src/pages/commissions/commission-page-view.njk

```diff
@@ -34,6 +34,12 @@
                         {{ commission.pageBody | safe }}
                     </div>
 
+                    {% if commission.slug == "ic-travel" %}
```

Comment:
I don't think this is a great way to do it:

- the commision page template should be as content-agnostic as possible (it is the template for all commissions)
- depending on the slug is also error-prone, it may desynchronize with the config of the IC Travel commission later

Why not put the iframe directly in the body of `ictravel.md`? As you can see above ` {{ commission.pageBody | safe }}` the content is assumed to be safe so the HTML will be rendered as is (and HTML in Markdown is mostly part of the spec https://www.markdownguide.org/basic-syntax/#html).

My recommendation is to remove this block entirely, and place the iframe at the bottom of https://github.com/clicepfl/clic-website/pull/172/files#diff-703fdb006e010b599a8ed8671c9816e0a4eeacf0e433cbeed26e09daefa0907e directly

Reply:
Oh indeed, I was not thinking about just putting HTML on Markdown but that's exactly what I wanted, lemme update that